JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -65,10 +67,22 @@ module Microsoft.ApplicationInsights {
         public session: Context.Session;
 
         /**
+        * Custom properties added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public properties;
+
+        /**
+        * Custom measurements added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public measurements;
+        
+        public onInitializeTelemetry: (envelope: Telemetry.Common.Envelope) => boolean;
+
```

Comment:
Since onInitialzeTelemetry not just initializes but also makes decisions about whether to include/exclude telemetry item, should the verb be "initialize"? Perhaps "process" or "transform" would be better verbs. When I think "initialize" I "remove" is not usually in the realm of possible things to happen during "initialization", but when I think "process" this may include "remove". onBeforeSendTelemetry would also be an interesting option.